nsist/__init__.py

```diff
@@ -197,6 +200,15 @@ def fetch_python_embeddable(self):
         with zipfile.ZipFile(str(cache_file)) as z:
             z.extractall(python_dir)
 
+        # Manipulate any *._pth files so the default paths AND pkgs directory
+        # ends up in sys.path.
+        pth_files = [f for f in os.listdir(python_dir)
```

Comment:
Pynsist 2 requires Python >= 3.5, so you could use [os.scandir()](https://docs.python.org/3/library/os.html) here if you want. It's not performance critical, though, so feel free to stick with `listdir()` if you'd rather.